views/profile.ejs

```diff
@@ -1,7 +1,15 @@
 <div class="ui container" id="profileDiv">
-  <h2>Profile Page</h2>
+  <h2>Meetings Calendar</h2>
+<div>You have <%= userMeetings.length %> meetings.</div>
 
-  <h3>Welcome to your PROFILE</h3>
+<h3></h3>
+<div class="hidden" id="userMeetings">
+  <% userMeetings.forEach(meeting => { %>
+    <div data-title="<%= meeting.dataValues.notes %>" data-date="<%= meeting.dataValues.dateTime %>"></div>
+    <% }) %>
+</div>
 
-  <!-- insert meetings statistics such as how many meetings they have, how many meetings they have of each categories, etc -->
+</div>
+<!-- insert meetings statistics such as how many meetings they have, how many meetings they have of each categories, etc -->
```

Comment:
starting to feel like feature creep, if you feel like you can get it done in the next day or two go for it, otherwise I'd say hold off and save for after the cohort, and lets focus on polishing everything up.